### PR TITLE
run test on jormungandr directory if we build only jormungandr and jcli

### PR DESCRIPTION
**Pull request by dkijania**

### CODE REVIEW

File: .github/workflows/ci_tests.yml [hunk 1]
```diff
@@ -41,16 +41,15 @@ jobs:
...
           toolchain: "1.65"  # it says it can read the rust-toolchain file, but it fails if we omit this
           components: rustfmt, clippy
       - run: |
-          cargo build -p jormungandr
-          cargo build -p jcli
-          cargo nextest run --no-fail-fast --partition hash:${{ matrix.shardNumber }}/${{ matrix.totalShards }}
-
+          cd src/jormungandr
+          cargo build
+          cargo nextest run -p jormungandr -p jcli -p jormungandr-integration-tests --no-fail-fast --partition hash:${{ matrix.shardNumber }}/${{ matrix.totalShards }}
```
Comment (cameron1024) [inline .github/workflows/ci_tests.yml:52]: This means we're not testing `vit-servicing-station` right? Or am I missing something

**Review by cameron1024 — APPROVED** [review]

**Comment by minikin** [comment]
@dkijania Can we merge?

**Comment by cameron1024** [comment]
I'm opening another PR to add back the missing tests now, fixed a couple of the CI failures so want to get them merged

File: .github/workflows/ci_tests.yml (hunk 1 above)
Comment (dkijania) [reply .github/workflows/ci_tests.yml]: well I need to confess that this is experimental change to narrow down the issues with insufficient space. I need to add other tests in different jobs

**Comment by minikin** [comment]
@dkijania Maybe, then next time we can use the draft pr option on GitHub and label it `do not review yet` :)

**Comment by dkijania** [comment]
Yes sorry, I was quite sure that tests are not running on draft pr

**Comment by cameron1024** [comment]
I've had mixed results with this, sometimes it seems like tests don't run on drafts, but I think it's also that tests don't run on PRs with merge conflicts

**Comment by minikin** [comment]
@dkijania @cameron1024 
We also use something like [DO NOT MERGE] or [DO NOT REVIEW] on the PR title to eliminate future confusion.
e.g:
```txt
[DO NOT REVIEW] run test on jormungandr directory if we build only jormungandr and jcli
```

**Pull request merged**